relay-general/src/store/normalize/contexts.rs

```diff
@@ -69,18 +72,34 @@ fn normalize_runtime_context(runtime: &mut RuntimeContext) {
     }
 }
 
+fn get_windows_version(description: &str) -> Option<&str> {
+    if let Some(captures) = OS_WINDOWS_REGEX1.captures(description) {
+        captures.name("version").map(|m| m.as_str())
+    } else if let Some(captures) = OS_WINDOWS_REGEX2.captures(description) {
+        captures.name("version").map(|m| m.as_str())
+    } else {
+        None
+    }
+}
+
+#[allow(dead_code)]
+pub fn get_android_api_version(description: &str) -> Option<&str> {
```

Comment:
Let's leave a note why this code is inactive for now, maybe a TODO comment.

Reply:
ok